packages/babel-plugin-emotion/package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "babel-generator": "^6.26.0",
+    "babel-helper-module-imports": "^7.0.0-beta.3",
```

Comment:
this is also published already as scoped - `@babel/helper-module-imports`

you probably should update other babel dependencies to their respective` @babel/*@7`